readme.md

```diff
@@ -135,6 +135,37 @@ const tasks = new Listr([
 It's also possible to return a `stream`. The stream will be converted to an `Observable` and handled as such.
 
 
+#### Skipping tasks
+
+<img src="media/skipped.png" width="255" align="right">
+
+At any point during execution a task can be skipped by invoking the `skip()` function on the task instance passed in to the task function.
+
+```js
+const tasks = new Listr([
+	{
+		title: 'Task 1',
+		task: () => Promise.resolve('Foo')
+	},
+	{
+		title: 'Can be skipped',
+		task: task => {
+			if (someCondition) {
+				task.skip('Reason for skipping');
```

Comment:
>  If you think returning a symbol is the better option I'm happy to make the change.

Let's see what @jamestalmage and @SamVerschueren thinks first. I might be wrong.

> Also there is a comment (2 lines down)

Yes, that was my point. I'd rather have explicit code than a comment explaining the code.